examples/api-deploy-gke-nebius-okta/README.md

```diff
@@ -0,0 +1,296 @@
+
+<div align="center">
+  <img src="https://i.imgur.com/k17TpJU.png" width="500px" alt="Okta login page">
+</div>
+
+If the login page shows 503 error, make sure the API server pod is healthy:
+```bash
+kubectl get pods --namespace $NAMESPACE --kubeconfig $TMP_KUBECONFIG --context $GKE_CONTEXT
+```
+
+## Step 4: Configure the CLI and launch your first job
+
+On your client(s), install the SkyPilot CLI:
+```bash
+pip install -U skypilot-nightly
+```
+
+Login to the API server:
+```bash
+sky api login -e $ENDPOINT # E.g., http://34.42.25.204 or http://sky.yourorg.com
+```
+
+A browser will open and you will be redirected to the Okta login page. Login with your Okta credentials. You will receive a token:
+
+<div align="center">
+  <img src="https://i.imgur.com/OflAVTp.png" width="500px" alt="Okta token page">
+</div>
+Copy the token and paste it in the CLI. You should see the following message:
+
+```console
+$ sky api login -e http://sky.yourorg.com
+Authentication is needed. Please visit this URL setup up the token:
+
+http://sky.yourorg.com/token
+
+Opening browser...
+Paste the token: 
+```
+
+Run `sky check` to verify cloud setup:
+```console
+$ sky check
+...
+🎉 Enabled infra 🎉
+  GCP [compute, storage]
+  Kubernetes [compute]
+    Allowed contexts:
+    ├── nebius-cluster
+    └── gke-cluster
+  Nebius [compute]
+```
+
+## 🎉 SkyPilot API server is ready to use!
+
+Some commands to try:
+
+* `sky dashboard` to open the dashboard in your browser
+* `sky launch -c test --gpus H100:1 -- nvidia-smi` to launch a job with 1 H100 GPU
+* `sky show-gpus` to show available GPUs
+* `sky status` to see SkyPilot status and infra available
+
+## ✨ Bonus: Use Nebius shared filesystem with SkyPilot
+
+You can also use Nebius shared filesystem with SkyPilot to get high performance data storage for datasets, checkpoints and more. 
+
+When creating a node group on the Nebius console, simply attach your desired shared file system to the node group (``Create Node Group`` -> ``Attach shared filesystem``):
+
+* Ensure ``Auto mount`` is enabled.
+* Note the ``Mount tag`` (e.g. ``filesystem-d0``).
+
+<div align="center">
+  <img src="https://i.imgur.com/02PhLB5.png" width="50%" alt="Nebius shared filesystem">
+</div>
+
+Nebius will automatically mount the shared filesystem to hosts in the node group. You can then use a ``hostPath`` volume to mount the shared filesystem to your SkyPilot pods.
+
+Here's an example of how to use the shared filesystem in a SkyPilot job:
+
+```yaml
+resources:
+  infra: k8s/nebius-mk8s-nebius-gpu-dev
+
+run: |
+  echo "Hello, world!" > /mnt/nfs/hello.txt
+  ls -la /mnt/nfs
+
+config:
+  kubernetes:
+    pod_config:
+      spec:
+        containers:
+          - volumeMounts:
+              - mountPath: /mnt/nfs
+                name: nebius-sharedfs
+        volumes:
+          - name: nebius-sharedfs
+            hostPath:
+              path: /mnt/<mount_tag> # e.g. /mnt/filesystem-d0
+              type: Directory
```

Comment:
Should we also mention that this can be configured in the global config with the dashboard, so that the NFS got mounted to all SkyPilot clusters launched?

Reply:
Added!